astropy/io/votable/dataorigin.py

```diff
@@ -0,0 +1,514 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Extract Data Origin in VOTable
```

Comment:
This doesn't show up anywhere in the docs, so I would recommend ditching it altogether in favour of narrative documentation and docstrings in the classes/functions.